mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -1152,4 +1173,33 @@ class GeneralizedLinearRegressionTrainingSummary private[regression] (
         "No p-value available for this GeneralizedLinearRegressionModel")
     }
   }
+
+  /**
+   * Summary table with feature name, coefficient, standard error,
+   * tValue and pValue.
+   */
+  @Since("2.2.0")
+  lazy val summaryTable: DataFrame = {
+    if (isNormalSolver) {
+      var featureNamesLocal = featureNames
+      var coefficients = model.coefficients.toArray
+      var idx = Array.range(0, coefficients.length)
+      if (model.getFitIntercept) {
+        featureNamesLocal = featureNamesLocal :+ Intercept
+        coefficients = coefficients :+ model.intercept
+        // Reorder so that intercept comes first
+        idx = (coefficients.length - 1) +: idx
+      }
+      val result = for (i <- idx.toSeq) yield
+        (featureNamesLocal(i), coefficients(i), coefficientStandardErrors(i),
+        tValues(i), pValues(i))
+
+      val spark = SparkSession.builder().getOrCreate()
```

Comment:
@felixcheung Would you elaborate on your concern here? The reason I create the SparkSession is to use convert `Seq` to `DataFrame` using `toDF`. Is there a way we can create data frame without explicitly using spark session? Thanks.